index.html

```diff
@@ -58,107 +59,120 @@
+            <svg class="offer__icon offer__icon--reviews">
+              <use href="images/sprites/icons.svg#reviews"></use>
+            </svg>
+            <p class="offer__desc">Real guest reviews</p>
+          </div>
         </div>
       </div>
     </section>
-  <div class="footer">
-    <div class="footer__container">
-      <ul class="footer-item">
-        <li class="footer-item__title">About</li>
-        <li><a href="#" class="footer-item__link">How Triphouse works</a></li>
-        <li><a href="#" class="footer-item__link">Careers</a></li>
-        <li><a href="#" class="footer-item__link">Privacy</a></li>
-        <li><a href="#" class="footer-item__link">Terms</a></li>
-      </ul>
-      <ul class="footer-item">
-        <li class="footer-item__title">Property types</li>
-        <li><a href="#" class="footer-item__link">Guest houses</a></li>
-        <li><a href="#" class="footer-item__link">Hotels</a></li>
-        <li><a href="#" class="footer-item__link">Apartments</a></li>
-        <li><a href="#" class="footer-item__link">Villas</a></li>
-        <li><a href="#" class="footer-item__link">Holiday homes</a></li>
-        <li><a href="#" class="footer-item__link">Hostels</a></li>
-      </ul>
-      <ul class="footer-item">
-        <li class="footer-item__title">Support</li>
-        <li><a href="#" class="footer-item__link">Contact Customer Service</a></li>
-        <li><a href="#" class="footer-item__link">FAQ</a></li>
-      </ul>
+  <footer class="footer">
+    <div class="footer__wrapper wrapper">
+      <svg class="footer__icon footer__icon--white icon--hidden">
+        <use href="images/sprites/icons.svg#logo"></use>
+      </svg>
+      <div class="footer__block col-sm-4">
+        <ul class="footer__list col-lg-4">
+          <li class="footer__title">About</li>
+          <li><a href="#" class="footer__link">How Triphouse works</a></li>
+          <li><a href="#" class="footer__link">Careers</a></li>
+          <li><a href="#" class="footer__link">Privacy</a></li>
+          <li><a href="#" class="footer__link">Terms</a></li>
+        </ul>
+        <ul class="footer__list col-lg-4">
+          <li class="footer__title">Property types</li>
+          <li><a href="#" class="footer__link">Guest houses</a></li>
+          <li><a href="#" class="footer__link">Hotels</a></li>
+          <li><a href="#" class="footer__link">Apartments</a></li>
+          <li><a href="#" class="footer__link">Villas</a></li>
+          <li><a href="#" class="footer__link">Holiday homes</a></li>
+          <li><a href="#" class="footer__link">Hostels</a></li>
+        </ul>
+        <ul class="footer__list col-lg-4">
+          <li class="footer__title">Support</li>
+          <li><a href="#" class="footer__link">Contact Customer Service</a></li>
+          <li><a href="#" class="footer__link">FAQ</a></li>
+        </ul>
+      </div>
+      <p class="footer__copyright col-sm-4">© 2022 Triphouse, Inc. All rights reserved</p>
```

Comment:
Good homework! But in adaptive design this text stay at one  place

Reply:
fixed that with margin: 0 auto